src/pages/events/createEventsPageComponenets/CreateEvetsStepper.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import { StepIconProps } from '@mui/material/StepIcon';
+import ColorlibStepIconRoot from "../createEvents/stepper/ColorlibStepIconRoot";
+import useCreateEventStyles from "../../../assets/styleJs/events/createEvent";
+import ColorlibConnector from "../createEvents/stepper/ColorlibConnector";
+
+const ColorlibStepIcon=({active, className, completed, icon}: StepIconProps) =>{
+
+  const icons: { [index: string]: string } = {
+    1: '1',
+    2: '2',
+    3: '3',
+  };
+
+  return (
+    <ColorlibStepIconRoot ownerState={{ completed, active }} className={className}>
+      {icons[String(icon)]}
+    </ColorlibStepIconRoot>
+  );
+}
+const CustomizedSteppers = () => {
+  const classes = useCreateEventStyles()
+
+  return (
+    <Stack className={classes.stepperStack}>
+
+      <Stepper activeStep={1} connector={<ColorlibConnector/>} className={classes.stepperRoot}>
+        {[1, 2, 3].map((label) => (
+          <Step key={label} className={classes.stepperStep}>
+            <StepLabel className={classes.stepperStepLabel} StepIconComponent={ColorlibStepIcon}/>
+          </Step>
+        ))}
```

Comment:
do not write map this way use variable for map on it

Reply:
done

Reply:
not fixed